ui/src/views/Kubernetes/index.vue

```diff
@@ -51,72 +46,73 @@ import { TreeOption } from 'naive-ui';
 import mittBus from '@/utils/mittBus';
 import Tree from '@/components/Kubernetes/Tree/index.vue';
 import SideTop from '@/components/Kubernetes/Sidebar/sideTop.vue';
-import MainContent from '@/components/Kubernetes/MainContent/index.vue';
+import MainContent from '@/components/Kubernetes/MainContent/main.vue';
 import ContentHeader from '@/components/Kubernetes/ContentHeader/index.vue';
+import { useI18n } from 'vue-i18n';
 
 const socket = ref();
 const sideWidth = ref(300);
 const isFolded = ref(false);
 
-const treeStore = useTreeStore();
-const { createTreeConnect } = useK8s();
+const { t } = useI18n();
 
-socket.value = createTreeConnect();
+const treeStore = useTreeStore();
+socket.value = useKubernetes(t);
 
 /**
  * 加载节点
  *
  * @param _node
  */
 const handleSyncLoad = (_node?: TreeOption) => {
-    // syncLoadNodes(node);
+  // syncLoadNodes(node);
 
-    // 根据节点宽度自动拓宽
-    setTimeout(() => {
-        const tableElement = document.querySelector('.n-descriptions-table') as HTMLElement;
-        const sideElement = document.querySelector('.n-layout-sider') as HTMLElement;
+  // 根据节点宽度自动拓宽
+  setTimeout(() => {
+    const tableElement = document.querySelector('.n-descriptions-table') as HTMLElement;
+    const sideElement = document.querySelector('.n-layout-sider') as HTMLElement;
 
-        if (tableElement && sideElement) {
-            const tableWidth = tableElement.clientWidth;
+    if (tableElement && sideElement) {
+      const tableWidth = tableElement.clientWidth;
 
-            sideWidth.value = tableWidth;
-            sideElement.style.width = `${tableWidth}px`;
-        }
-    }, 300);
+      sideWidth.value = tableWidth;
+      sideElement.style.width = `${tableWidth}px`;
+    }
+  }, 300);
 };
 
 /**
  * 点击 Tree 图标的回调
  */
 const handleTreeClick = () => {
-    isFolded.value = !isFolded.value;
-    sideWidth.value = isFolded.value ? 0 : 300;
+  isFolded.value = !isFolded.value;
+  sideWidth.value = isFolded.value ? 0 : 300;
 };
 
 /**
  * 重新加载
  */
 const handleReloadTree = () => {
-    if (socket.value) {
-        treeStore.setReload();
-        socket.value.send(JSON.stringify({ type: 'TERMINAL_K8S_TREE' }));
-    }
+  if (socket.value) {
+    treeStore.setReload();
+    socket.value.send(JSON.stringify({ type: 'TERMINAL_K8S_TREE' }));
+  }
 };
 
 const handleDragEnd = (_el: HTMLElement, newWidth: number) => {
-    const tableElement = document.querySelector('.n-collapse') as HTMLElement;
+  const tableElement = document.querySelector('.n-collapse') as HTMLElement;
 
-    nextTick(() => {
-        tableElement.style.width = newWidth + 'px';
-    });
+  nextTick(() => {
+    tableElement.style.width = newWidth + 'px';
+  });
 };
 
 onMounted(() => {
-    mittBus.on('fold-tree-click', handleTreeClick);
+  mittBus.on('fold-tree-click', handleTreeClick);
 });
 
 onUnmounted(() => {
-    mittBus.off('fold-tree-click', handleTreeClick);
+  mittBus.off('fold-tree-click', handleTreeClick);
 });
 </script>
 
```

Comment:
The changes include:

- The layout for sidebar content (`<n-layout>`) and other elements have been merged into an overall layout component named `CustomLayout`, which makes better usage of space.

- Layout styles (width, max width etc.) are more consistent with CSS to maintain best practices instead of inline JavaScript. 

Optimization Suggestions:
1. Utilize computed properties effectively: Use `style` where necessary to avoid multiple style updates at once using `v-bind`.
2. Extract common logic out of functions to reduce complexity and improve readability.
3. Avoid overreliance on inline scripts that could potentially conflict or become outdated when shared between pages. Consider moving them outside the script blocks to scoped components like Vuex store modules if they are used throughout many pages.
   
Remember, all the above suggestions can be tailored based on specific project contexts.